pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -606,6 +606,11 @@ func (m *volumeMounter) getSourcePodFilePath(sourceUID types.UID, vmi *v1.Virtua
 		}
 		for _, mount := range mounts {
 			if mount.MountPoint == "/pvc" {
+				for _, prefix := range m.stripPaths() {
+					if strings.HasPrefix(mount.Root, prefix) {
+						return strings.TrimPrefix(mount.Root, prefix), nil
+					}
+				}
```

Comment:
is there some other mountPoint we should be looking for rather than stripping the prefix?

Reply:
I am not particularly fond of stripping it either. I just realized I stop on the first one mount.MountPoint == "/pvc" there might be more. Let me investigate that some.

Reply:
This is really just a wild speculation, but is this PR related here: https://github.com/kubevirt/kubevirt/pull/5108? Maybe something to borrow from there? I did not check this PR in detail, so feel free to ignore if irrelevant.

Reply:
No, I use the GetProcMounts from github.com/prometheus/procfs, and that just reads from /proc/<PID>/mountinfo. The value is wrong in that file for RHCOS based nodes. In those nodes the mountinfo file contains /ostree/deploy/rhcos/<actual path I want>. I talked to the RHCOS guys, and they told me it has something to do with how ostree/ostree-rpm works and that it would be relatively save to just chop off the prefix as that should never change.